openbb_terminal/cryptocurrency/due_diligence/dd_controller.py

```diff
@@ -32,7 +32,7 @@
     cryptopanic_view,
     tokenterminal_view,
 )
-from openbb_terminal.decorators import log_start_end
```

Comment:
The `check_api_key` is not being used. Did you want it to be uncommented?

Reply:
oh oops, we had it there, i'll fix